run_Docker.sh

```diff
@@ -98,7 +98,7 @@ To install on Alma Linux please run:\e[0m
 		-v ${PWD}/Gui/QtGUIutils/PeltierCoolingApp.py:/home/cmsTkUser/Ph2_ACF_GUI/Gui/QtGUIutils/PeltierCoolingApp.py\
         -v ${PWD}/Gui/python/Peltier.py:/home/cmsTkUser/Ph2_ACF_GUI/Gui/python/Peltier.py\
 		-w $PWD  -e DISPLAY=$DISPLAY\
-		--volume="$HOME/.Xauthority:/root/.Xauthority:rw" --net host majoyce2/ph2_acf_gui_user:latest #local/testimagejuly30user
+		--volume="$HOME/.Xauthority:/root/.Xauthority:rw" --net host osupixels/ph2_acf_gui_user:latest #local/testimagejuly30user
```

Comment:
We should still pull the image from the majoyce2 repository by default. Also I've found that using -v /tmp/.X11-unix:/tmp/.X11-unix works better to display graphics. 
```suggestion
		--volume="$HOME/.Xauthority:/root/.Xauthority:rw" --net host osupixels/ph2_acf_gui_user:latest #local/testimagejuly30user
```